node/src/components/network.rs

```diff
@@ -8,10 +8,11 @@ mod peer_discovery;
 mod protocol_id;
 #[cfg(test)]
 mod tests;
+#[cfg(test)]
+mod tests2;
```

Comment:
I guess a more meaningful name would be good here.  Or would renaming the old module be more appropriate?

Reply:
Been renamed!